kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -89,6 +97,11 @@ window.LoginView = BaseView.extend({
     handle_login: function(response) {
         if (response.status == 200) {
             this.trigger("login_success");
```

Comment:
Because the modal logic is separated from the interior login view.

On Tue, 26 May 2015 13:59 Michael Gallaspy <notifications@github.com> wrote:

> In kalite/distributed/static/js/distributed/user/views.js
> <https://github.com/learningequality/ka-lite/pull/3729#discussion_r31079215>
> :
>
> > @@ -89,6 +97,11 @@ window.LoginView = BaseView.extend({
> >      handle_login: function(response) {
> >          if (response.status == 200) {
> >              this.trigger("login_success");
>
>  I don't know how this could possibly result in an unsafe expectation.
>
> It doesn't right now, but my point is that it could in the future if
> someone decides to add another listener. Since this *could* trigger a
> page load (yet *may* not), any listener will have to finish what it's
> doing in a hurry or potentially be cut short. If all you wish to do is
> close the modal, why not just call close_modal directly?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learningequality/ka-lite/pull/3729/files#r31079215>.
>